[2025-09-25T08:23:01.170+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-25T08:23:01.191+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_test_dag.bash_hello scheduled__2025-09-25T08:15:00+00:00 [queued]>
[2025-09-25T08:23:01.196+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_test_dag.bash_hello scheduled__2025-09-25T08:15:00+00:00 [queued]>
[2025-09-25T08:23:01.196+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-25T08:23:01.205+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): bash_hello> on 2025-09-25 08:15:00+00:00
[2025-09-25T08:23:01.209+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'simple_test_dag', 'bash_hello', 'scheduled__2025-09-25T08:15:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/simple_test_dag.py', '--cfg-path', '/tmp/tmpmlodyhn_']
[2025-09-25T08:23:01.211+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask bash_hello
[2025-09-25T08:23:01.210+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=207) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-25T08:23:01.212+0000] {standard_task_runner.py:64} INFO - Started process 208 to run task
[2025-09-25T08:23:01.236+0000] {task_command.py:426} INFO - Running <TaskInstance: simple_test_dag.bash_hello scheduled__2025-09-25T08:15:00+00:00 [running]> on host ee3253342b48
[2025-09-25T08:23:01.281+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='simple_test_dag' AIRFLOW_CTX_TASK_ID='bash_hello' AIRFLOW_CTX_EXECUTION_DATE='2025-09-25T08:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-25T08:15:00+00:00'
[2025-09-25T08:23:01.281+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-25T08:23:01.291+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-25T08:23:01.291+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Hello from Bash!' && date"]
[2025-09-25T08:23:01.296+0000] {subprocess.py:86} INFO - Output:
[2025-09-25T08:23:01.299+0000] {subprocess.py:93} INFO - Hello from Bash!
[2025-09-25T08:23:01.301+0000] {subprocess.py:93} INFO - Thu Sep 25 08:23:01 UTC 2025
[2025-09-25T08:23:01.302+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-25T08:23:01.302+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-25T08:23:01.315+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_test_dag, task_id=bash_hello, run_id=scheduled__2025-09-25T08:15:00+00:00, execution_date=20250925T081500, start_date=20250925T082301, end_date=20250925T082301
[2025-09-25T08:23:01.356+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-25T08:23:01.372+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-25T08:23:01.376+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
